dao/exec.go

```diff
@@ -144,3 +158,16 @@ func (l *ListExec) Join(execs []Executable) []Executable {
 	join = append(join, execs...)
 	return join
 }
+
+//-----------------------------------------------------------------------------
+// Result Helpers
+//-----------------------------------------------------------------------------
+
+func RowsAffected(result Result) int64 {
+	if result != nil {
+		if rows, err := result.RowsAffected(); err == nil {
```

Comment:
Should we be logging this error?

Reply:
This error is SQL driver related. Some drivers do not implement this method and returns `not implemented` error. It does not relate to logic/execution failures at all.